app/assets/stylesheets/content/_select2.scss

```diff
@@ -200,3 +186,7 @@ $se2-width: 100%;
     }
   }
 }
+
+.select2-input {
+  height: rem-calc($se2-input-height) !important;
```

Comment:
@myabc I know, just had hard time to rewrite foundation defaults. Let me try one more thing to get rid of ```!important```